docs/docs/guides/compatibility.mdx

```diff
@@ -0,0 +1,29 @@
+---
+id: compatibility
+title: Compatibility
+sidebar_label: Compatibility
+---
+
+# Compatibility Table
+
+In order to provide solid, state-of-the arts solutions we often use brand new React Native features,
+that are being introduced all the time. Thats why applications with mismatched Reanimated and React Native
+versions may show unexpected behavior or even fail to build at all. The table below contains official list
+of supported React Native versions.
+
+#### Supported React Native versions - New Architecture (Fabric)
+
+| Reanimated version(s)       | The oldest supported React Native | The newest supported React Native |
+| --------------------------- | --------------------------------- | --------------------------------- |
+| from **3.1.x** to **3.5.x** | 0.72                              | 0.72                              |
```

Comment:
This is not true

Reply:
Ok, actually on Fabric Reanimated 3.0.x supports RN 0.71 and 3.1.x-3.5.x support RN 0.72.